scripts/lint

```diff
@@ -61,7 +61,7 @@ def run() -> None:
         [
             "frontend/node_modules/.bin/prettier",
             "--config",
-            "frontend/package.json",
+            "frontend/prettier.config.js",
```

Comment:
Add this in a separate commit, where this change comes with all the files linted correctly.

Reply:
yeah makes sense